app/views/casa_cases/show.html.erb

```diff
@@ -124,9 +124,3 @@
     </div>
   </div>
 </div>
-
-<script>
-$(function () {
-  $('[data-toggle="tooltip"]').tooltip()
-})
-</script>
```

Comment:
The `$` was causing cypress to fail locally. So I have extracted this into it's own file. Maybe this will also fix the cypress failure on master.